docs/rhoas/iac_tools_rhoas/README.adoc

```diff
@@ -0,0 +1,686 @@
+`operation_type`:: The type of operation (an action performed on a resource) that is allowed for the given user on this module.
+`permission_type`:: Whether permission is given to the user or taken away.
+
+. To create a Kafka topic, copy the contents of the `create_kafka_topic` module shown in the following example and paste it into the `rhoas_kafka.yml` file.
++
+.Example `create_kafka_topic` module
+[source,yaml]
+----
+- name: Create Kafka Topic
+    create_kafka_topic:
+      topic_name: "kafka-topic-name"
+      kafka_id: "{{ kafka_req_resp.id }}"
+      # To hard code the kafka_id, uncomment and use the following line:
+      # kafka_id: "KAFKA_ID"
+      partitions: 1
+      retention_period_ms: "86400000"
+      retention_size_bytes: "1073741824"
+      cleanup_policy: "compact"
+      openshift_offline_token: "OPENSHIFT_OFFLINE_TOKEN"
+    register:
+      create_topic_res_obj
+----
+. You can allow Ansible to get the Kafka ID from the `kafka_req_resp.id` variable or enter it in the `kafka_id` field.
+. In the `openshift_offline_token` field, enter your OpenShift offline token.
+. To update the configuration of the topic, copy the `update_kafka_topic` module shown in the following example and paste it into the `rhoas_kafka` file. In the following example, the cleanup policy has been updated from compact to delete by replacing `"compact"` with `"delete"` in the `cleanup_policy` field.
++
+.Example `update_kafka_topic` module
+[source,yaml]
+----
+- name: Update Kafka Topic
+    update_kafka_topic:
+      topic_name: "kafka-topic-name"
+      kafka_id: "{{ kafka_req_resp.id }}"
+      # To hard code the kafka_id, uncomment and use the following line:
+      # kafka_id: "KAFKA_ID"
+      partitions: 1
+      retention_period_ms: "86400000"
+      retention_size_bytes: "1073741824"
+      cleanup_policy: "delete"
+      openshift_offline_token: "OPENSHIFT_OFFLINE_TOKEN"
+    register:
+      update_topic_res_obj
+----
+. You can allow Ansible to get the Kafka ID from the `kafka_req_resp.id` variable or enter it in the `kafka_id` field.
+. (Optional) You can modify the values in the `retention_period_ms` and `retention_size_bytes` fields instead of accepting the default values.
+. To delete the topic, copy the `delete_kafka_topic` module shown in the following example and paste it into the `rhoas_kafka.yml` file.
++
+.Example `delete_kafka_topic` module
+[source,yaml]
+----
+- name: Delete Kafka Topic
+   rhoas.rhoas.delete_kafka_topic:
+     topic_name: "KAFKA_TOPIC_NAME"
+      kafka_id: "{{ kafka_req_resp_obj['kafka_id'] }}"
+      # To hard code the kafka_id, uncomment and use the following line:
+      # kafka_id: "KAFKA_ID"
+     openshift_offline_token: "OPENSHIFT_OFFLINE_TOKEN"
+----
+. In the `openshift_offline_token` field, enter your OpenShift offline token.
+. You can allow Ansible to get the Kafka ID from the `kafka_req_resp.id` variable or enter it in the `kafka_id` field.
```

Comment:
In general, I would rewrite these "you can" steps because they make it a bit unclear whether the steps are required or not. There's a few of them in this doc. For example:

"To directly specify a Kafka instance ID, enter a value in the `kafka_id` field. Otherwise, Ansible gets the Kafka ID from the  `kafka_req_resp.id` variable."